### PR TITLE
Feature/blobfish docker

### PR DESCRIPTION
**Pull request by jemten**
### This PR fixes:

- see above

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: templates/mip_install_rd_rna_config_-1.0-.yaml [hunk 1]
```diff
@@ -41,6 +28,10 @@ singularity:
       arriba: /arriba_v1.1.0/arriba
       draw_fusions.R: /arriba_v1.1.0/draw_fusions.R
     uri: docker://uhrigs/arriba:1.1.0
+  blobfish:
+    executable:
+      BlobFish.py:
```
Comment (henrikstranneheim) [inline templates/mip_install_rd_rna_config_-1.0-.yaml]: What about: BlobFish.R

Comment (jemten) [reply templates/mip_install_rd_rna_config_-1.0-.yaml]: Thats also an executable in Blobfish, which we never use (I think that it is used internally in Blobfish.py though). That is why I didn't link it

Comment (henrikstranneheim) [reply templates/mip_install_rd_rna_config_-1.0-.yaml]: 👍

**Comment by jemten** [comment]
i'll fix the coverage. just have to delete the old install script for blobfish

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**